validator/build.py

```diff
@@ -252,37 +252,47 @@ def GenerateValidateBin(out_dir, nodejs_cmd):
 
       function main() {
         if (process.argv.length < 3) {
-          console.error('usage: validate <file.html or url>');
+          console.error('usage: validate (file.html || url || -)');
```

Comment:
I think the "syntax" used here was vaguely like in Unix. I think we should stick to it unless there's some other standard that people are familiar with. I don't know the NodeJS conventions, perhaps using || is a thing? But if not we should avoid being inventive. Here's some reference I could find off of hackernews.
http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html
Interpreting that, I think usage: `validate <file.html>|<url>|-`  might be as close as possible?